template/cordova/lib/AppxManifest.js

```diff
@@ -374,6 +374,11 @@ AppxManifest.prototype.getCapabilities = function () {
 };
 
 function refineColor(color) {
+    if (color.indexOf('0x') === -1 && color.indexOf('#') === -1) {
```

Comment:
Please put this condition into a function; that way it will be easier to debug and read in the future.